app/subscriber/src/components/sidebar/hooks/useFilterOptions.ts

```diff
@@ -1,64 +1,88 @@
 import React from 'react';
 import { toast } from 'react-toastify';
 import { useLookup } from 'store/hooks';
+import { ISourceModel, Settings } from 'tno-core';
 
-export const useFilterOptions = () => {
-  const [{ sources, settings }] = useLookup();
-  const [printIds, setPrintIds] = React.useState<{
-    dailyPrintId: number;
-    weeklyPrintId: number;
-    onlinePrintId: number;
-  }>();
-  const [mediaIds, setMediaIds] = React.useState<{
-    cpWireId: number;
-    talkRadioId: number;
-    newsRadioId: number;
-    televisionId: number;
-  }>();
+export interface IMediaTypeGroupIds {
+  cpWireId: number;
+  talkRadioId: number;
+  newsRadioId: number;
+  televisionId: number;
+  dailyPrintId: number;
+  weeklyPrintId: number;
+  onlinePrintId: number;
+}
+
+export interface IMediaTypeGroups {
+  dailyPrint: ISourceModel[];
+  weeklyPrint: ISourceModel[];
+  cpWire: ISourceModel[];
+  talkRadio: ISourceModel[];
+  onlinePrint: ISourceModel[];
+  newsRadio: ISourceModel[];
+  television: ISourceModel[];
+  sources: ISourceModel[];
+}
+
+const defaultMediaTypeGroups: IMediaTypeGroups = {
+  dailyPrint: [],
+  weeklyPrint: [],
+  cpWire: [],
+  talkRadio: [],
+  onlinePrint: [],
+  newsRadio: [],
+  television: [],
+  sources: [],
+};
+
+export const useFilterOptions = (): IMediaTypeGroups => {
+  const [{ isReady, sources, settings }] = useLookup();
+  const [mediaTypeGroups, setMediaTypeGroups] = React.useState<IMediaTypeGroups>({
```

Comment:
@jdtoombs We don't need two separate keys when one will do the job and simplify.  Also only store the values we need to keep in state.